src/ng-serializer.service.ts

```diff
@@ -1,6 +1,9 @@
 import { Injectable } from '@angular/core';
-import { Serializer } from '@kaiu/serializer';
+import { Serializer, Registry } from '@kaiu/serializer';
 
 @Injectable()
 export class NgSerializerService extends Serializer {
+    constructor(public readonly registry: Registry) {
```

Comment:
This has been done because else aot can't resolve all parameters for this service.